diff --git a/node_modules/react-native-sensors/android/.gradle/9.2.0/checksums/checksums.lock b/node_modules/react-native-sensors/android/.gradle/9.2.0/checksums/checksums.lock
new file mode 100644
index 0000000..e1d0448
Binary files /dev/null and b/node_modules/react-native-sensors/android/.gradle/9.2.0/checksums/checksums.lock differ
diff --git a/node_modules/react-native-sensors/android/.gradle/9.2.0/fileChanges/last-build.bin b/node_modules/react-native-sensors/android/.gradle/9.2.0/fileChanges/last-build.bin
new file mode 100644
index 0000000..f76dd23
Binary files /dev/null and b/node_modules/react-native-sensors/android/.gradle/9.2.0/fileChanges/last-build.bin differ
diff --git a/node_modules/react-native-sensors/android/.gradle/9.2.0/fileHashes/fileHashes.bin b/node_modules/react-native-sensors/android/.gradle/9.2.0/fileHashes/fileHashes.bin
new file mode 100644
index 0000000..094cd86
Binary files /dev/null and b/node_modules/react-native-sensors/android/.gradle/9.2.0/fileHashes/fileHashes.bin differ
diff --git a/node_modules/react-native-sensors/android/.gradle/9.2.0/fileHashes/fileHashes.lock b/node_modules/react-native-sensors/android/.gradle/9.2.0/fileHashes/fileHashes.lock
new file mode 100644
index 0000000..bffa225
Binary files /dev/null and b/node_modules/react-native-sensors/android/.gradle/9.2.0/fileHashes/fileHashes.lock differ
diff --git a/node_modules/react-native-sensors/android/.gradle/9.2.0/gc.properties b/node_modules/react-native-sensors/android/.gradle/9.2.0/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-sensors/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock b/node_modules/react-native-sensors/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock
new file mode 100644
index 0000000..2cfd1f9
Binary files /dev/null and b/node_modules/react-native-sensors/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock differ
diff --git a/node_modules/react-native-sensors/android/.gradle/buildOutputCleanup/cache.properties b/node_modules/react-native-sensors/android/.gradle/buildOutputCleanup/cache.properties
new file mode 100644
index 0000000..07b157b
--- /dev/null
+++ b/node_modules/react-native-sensors/android/.gradle/buildOutputCleanup/cache.properties
@@ -0,0 +1,2 @@
+#Sat Aug 08 15:02:37 GST 2026
+gradle.version=9.2.0
diff --git a/node_modules/react-native-sensors/android/.gradle/vcs-1/gc.properties b/node_modules/react-native-sensors/android/.gradle/vcs-1/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-sensors/android/build.gradle b/node_modules/react-native-sensors/android/build.gradle
index f1526ec..e7ea93f 100644
--- a/node_modules/react-native-sensors/android/build.gradle
+++ b/node_modules/react-native-sensors/android/build.gradle
@@ -6,13 +6,12 @@ def safeExtGet(prop, fallback) {
 repositories {
   mavenCentral()
   google()
-  jcenter()
 }
 
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
